extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -43,6 +43,28 @@ public static FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem
     public Optional<String> defaultSchema = Optional.empty();
 
+    /**
+     * Sets a specific user that will Flyway will use to connect to the datasource.
+     * Allows to have a specific user that can apply DDL to the schema while the standard user used for the datasource may not.
+     */
+
+    @ConfigItem
+    public Optional<String> username = Optional.empty();
+
+    /**
+     * Set a password that will Flyway will use to connect to the datasource.
+     */
+
+    @ConfigItem
+    public Optional<String> password = Optional.empty();
+
+    /**
+     * Set a new jdbcurl that will Flyway will use to connect to the datasource.
+     */
+
+    @ConfigItem
+    public Optional<String> jdbcUrl = Optional.empty();
+
```

Comment:
I think it would be nice to have this under a `datasource` group.

Reply:
Sorry, could you explain this?